backend/bamboo/core/extensions.py

```diff
@@ -1,5 +1,11 @@
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
+from sqlalchemy.orm import DeclarativeBase
+
+
+class Base(DeclarativeBase):
```

Comment:
This seems to be unnecessary.

Reply:
It's recommended by the [Flask-SQLAlchemy docs](https://flask-sqlalchemy.palletsprojects.com/en/3.1.x/models/#initializing-the-base-class).

Reply:
Right, but I have checked there is no difference w/ or w/o it, I think the docs need get updates.